ui/components/app/transaction-detail/index.scss

```
@@ -15,4 +15,55 @@
       text-transform: uppercase;
     }
   }
+
+  &-edit-V2 {
+    margin-bottom: 10px;
+    display: flex;
+    align-items: baseline;
+    justify-content: flex-end;
+    padding-top: 20px;
+
+    button {
+      @include H7;
+
+      display: flex;
+      align-items: baseline;
+      color: $primary-1;
+      background: transparent;
+      border: 0;
+      padding-inline-end: 0;
+      white-space: pre;
+    }
+
+    i {
+      color: $primary-1;
+      margin-right: 2px;
```

This one might be tricky without any visuals but should probably be `4px`. There are exceptions to the rule if it breaks the layout.

```suggestion
      margin-right: 4px;
```